src/Tribe/Template.php

```diff
@@ -263,10 +263,10 @@ public function set_template_context_extract( $value = false ) {
 	 *
 	 * @return array
 	 */
-	public function merge_context( $context = array(), $file = null, $name = null ) {
+	public function merge_context( $context = [], $file = null, $name = null ) {
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 55d664d0dd4996c84210a1265175bb62</sub>